docs/RMRK/access/Ownable.md

```diff
@@ -63,7 +63,7 @@ Returns the address of the current owner.
 
 | Name | Type | Description |
 |---|---|---|
-| _0 | address | undefined |
```

Comment:
Found ~200 matches of `| undefined |` in dev. We need to check those, I'll add a task.

Reply:
Yeah, my thoughts exactly, I meant to take a look tomorrow and address them.

Reply:
Here's the task: https://app.clickup.com/t/865bge1xz